src/components/PayloadDetailsCard.js

```diff
@@ -217,7 +233,9 @@ function ExtrinsicPart({ label, fallback, value }) {
             ? renderMethodDetails()
```

Comment:
I find this type of structure particularly hard to read, can we use the good old if/else?

Reply:
fair enough, i don't mind either way, ill chang eit

Reply:
sorry @Tbaut unfortunately I think I'll keep this as it is actually :/